crypto/signer.go

```diff
@@ -35,10 +35,20 @@ var sigpbHashLookup = map[crypto.Hash]sigpb.DigitallySigned_HashAlgorithm{
 // Signer is responsible for signing log-related data and producing the appropriate
 // application specific signature objects.
 type Signer struct {
+	KeyID  int64
```

Comment:
BTW, is this used for anything other than diagnostics/logging?

Reply:
At the moment it's only used for logging to my knowledge. 
In the future when we have gossip, this will help gossipers identify which public key to use to verify `SignedLogRoot`s

Reply:
Now I understand better I wonder: should this be `KeyHint []byte` ?

Reply:
That would be more consistent. Done.